spec/models/memberships/switch_stammsektion_spec.rb

```diff
@@ -50,6 +50,12 @@ def create_role(key, role, owner: person, **attrs)
       expect(switch).to be_valid
     end
 
+    it "is valid with membership in different section active since today" do
+      create_role(:matterhorn_mitglieder, "Mitglied", start_on: Time.zone.today)
+      expect(switch).to be_valid
+      expect(switch).to be_valid
```

Comment:
For some reason calling `.delete` only fails when calling this method twice? In our request this method is called twice, resulting in this bug, to recreate it in this spec we have to call it twice...